views/login.html

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
question about angular: is there a way to have the equivalent of main.html so every page doesnt require you to have stuff outside the `<body>` tags, and login.html would just be the inner parts?